interfaces/apparmor/template.go

```diff
@@ -649,6 +649,10 @@ profile snap-update-ns.###SNAP_NAME### (attach_disconnected) {
   # sharing and propagates mount events outside of the snap namespace.
   audit deny mount -> /media,
 
+  # Commonly needed permissions for writable mimics.
+  /tmp/ r,
+  /tmp/.snap/{,**} rw,
```

Comment:
Why is that necessary in /tmp?

Reply:
The question was about "/tmp/.snap/{,**} rw," specifically.

Reply:
This is the path used by snap-update-ns to stash the state of a directory during the construction of a writable mimic.